smdebug/core/hook.py

```diff
@@ -437,11 +437,6 @@ def _initialize_writers(self, only_initialize_if_missing=False) -> None:
                     self.logger.info(f"Hook is writing from the hook with pid: {os.getpid()}\n")
                 else:
                     self.first_process = False
-                    self.logger.warn(
-                        f"Unsupported Distributed Training Strategy Detected.\n\
-                        Sagemaker-Debugger will only write from one process.\n\
-                        The process with pid: {os.getpid()} will not be writing any data. \n"
-                    )
```

Comment:
We should probably move this log to some other location where it is called fewer times instead of completely removing it?

Reply:
fixed this in the commit i just pushed